cli/src/program_v4.rs

```diff
@@ -171,9 +202,10 @@ impl ProgramV4SubCommands for App<'_, '_> {
                                     "Program authority [default: the default configured keypair]",
                                 ),
                         )
-                        .arg(Arg::with_name("use_rpc").long("use-rpc").help(
+                        .arg(Arg::with_name("use-rpc").long("use-rpc").help(
```

Comment:
Could someone be using this already?

Reply:
The first name is internal and the long name is what is user facing:
See https://docs.rs/clap/2.34.0/clap/struct.Arg.html